src/components/DeliverableView/DocumentsList.tsx

```diff
@@ -28,13 +28,23 @@ const DocumentsList = (props: DocumentsListProps): JSX.Element => {
       direction: order === 'asc' ? 'Ascending' : 'Descending',
     });
 
+  const rows = useMemo(
+    () =>
+      props.deliverable.documents.map((document) => ({
+        ...document,
+        projectName: props.deliverable.projectName,
+        deliverableId: props.deliverable.id,
+      })),
```

Comment:
I went this route as opposed to plumbing this redundant info into the BE response, let me know if you'd rather I go that route.

Reply:
This is totally fine. 
Ideally we create reducers to produce these views but overkill here.